### PR TITLE
Draft: started with upload of images

### PR DESCRIPTION
**Pull request by Filipe-Inacio**

### CODE REVIEW

File: marketplace-service/src/main/java/ch/akros/marketplace/service/controller/TopicController.java [hunk 1]
```diff
@@ -3,53 +3,58 @@
...
-    } catch (Exception ex) {
-      log.error(ex.getMessage(), ex);
-      return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
+    @PostMapping(value = "/topics", consumes = { "multipart/form-data" })
+    public ResponseEntity<Void> createTopic(@RequestPart("topics") String topicSaveRequestDTO,
+                                            @RequestPart(value = "thumbnail", required = false) MultipartFile thumbnail,
+                                            @RequestPart(value = "images", required = false) MultipartFile[] images) {
+        try {
+            log.debug("TopicController.topicsPost() called");
+            topicService.saveTopic(topicSaveRequestDTO, images, thumbnail);
```
Comment (pestadieu) [inline marketplace-service/src/main/java/ch/akros/marketplace/service/controller/TopicController.java:38]: Before saving the images, would it be possible to verify that the files uploaded are indeed images? Or maybe create an issue to deal with that later.

File: am-ui/webcomponents/create-ad-webcomponent/src/components/CreateTopic.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import ApiClient from "../api/src/ApiClient";
+import axios from 'axios'
```
Comment (pestadieu) [inline am-ui/webcomponents/create-ad-webcomponent/src/components/CreateTopic.js:2]: Missing ";" at the end of the line.

File: am-ui/src/app/app.module.ts [hunk 3]
```diff
@@ -92,6 +94,7 @@ export function MsalInterceptorConfigFactory(): MsalInterceptorConfiguration {
     SearchResultDetailsComponent,
     SearchResultDetailsMapComponent,
     BackButtonDirective,
+    SwiperComponent,
```
Comment (aschiakros) [inline am-ui/src/app/app.module.ts:97]: Ist dieses Komma ok?

Comment (Filipe-Inacio) [reply am-ui/src/app/app.module.ts:97]: Es hat immer ein Komma zum schluss und Lint meldet keinen Fehler. Daher denke ich, dass es passt :)

**Pull request merged**